README.md

```diff
@@ -47,11 +41,14 @@ There are several packages that makeup the Dojo 2 platform:
 
 We also have packages that are still in their planning stages and as such might not be part of the initial Dojo 2 release.
 
+* [`dojo/actions`](https://github.com/dojo/actions) - A command like library for Dojo 2 applicationsz
+* [`dojo/app`](https://github.com/dojo/app) - An application framework for Dojo 2
 * [`dojo/crypto`](https://github.com/dojo/crypto) - A set of libraries to deal with cryptography
+* [`dojo/dataviz`](https://github.com/dojo/dataviz) - A data vizualization library for Dojo 2
 
+We have some deprecated packages:
 
-We have deprecated a package:
-
+* [`dojo/dom`](https://github.com/dojo/dom) - A set of APIs for manipulating the DOM
```

Comment:
I think for now it is deprecated in it current form. It was written pre typescript 1.8 (even though it has been updated through to 2) and is not used anywhere within the current dojo 2 packages. Would be happy to revisit once we have some use cases?